src/commands/google.ts

```diff
@@ -71,11 +71,12 @@ const googleModule: CommandType = {
         let i: number = 0;
 
         if (data.items.length == 0) {
-          links.push("No results.");
+          links[0] = "No results.";
```

Comment:
Is this going to error out? The array doesn't contain any items so `[0]` is out of bounds.

Also, since you're using `map`, you can do something like
```js
let response;

if (no items) {
	response = "No results";
} else {
	response = items.map(...).join('\n')
}
```
or similar